### PR TITLE
3369 reverted unrelated changes 

### LINKED ISSUE #3369: Add Definitions of terms push and fetch under " Explanation About Repositories and Syncing Process"
### Please verify the following:

- [X] This issue is related to the [Virtual Interns First Steps](https://open-learning-exchange.github.io/#!pages/vi/vi-first-steps.md) document or [Planet User Manual](https://open-learning-exchange.github.io/#!pages/manual/planet/overview.md).
- [X] You have searched for relevant keywords in [recent open and closed issues](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues?q=is%3Aissue+label%3Avi) to avoid duplicates.

### URL to the MDwiki page containing the problem

https://open-learning-exchange.github.io/#!pages/vi/vi-github-and-repositories.md

### Problem

I noticed that there is a photo that outlines how to sync a fork. There are no definitions for the words of "fetch" and "push"

### Screenshots (when necessary)

updatedIssue

### Proposed solution(s)

I feel adding what these terms mean before presenting the photo of how the syncing process works will give the intern a better idea as to what fetch does and what push is actually doing

### PR DESCRIPTION
**Pull request by Gavinp14**
**Important:** *Do not tick a checkbox if you haven’t performed its action.*

### Before Creating the Pull Request

- [x] Verify the changes are on a new branch other than `master` – it should say "compare: branch-name".
- [x] Start the pull request title with lowercase letters, e.g., `add githubusername.md`.
- [x] Add issue number to the end of the pull request title when applicable, e.g., `update mi-faq.md (fixes #3264)`.

### After Creating the Pull Request

- [x] Go to "Commits" tab, make sure the commit username is clickable and linked to your GitHub account correctly.
- [x] Review the "Files changed" tab to ensure there are no unnecessary files or changes included in the pull request.
- [x] Verify that the raw.githack preview link is included in the description.
  - [ ] Preview the MDwiki rendered changes using the raw.githack link. Ensure it displays as expected without any errors.
- [ ] Drop a link to this pull request in our discord channel.

### Description, Screenshots and/or Screencast

Added definitions of push and pull in step 5 before presenting the sync fork process
change

fixes #3369 

https://raw.githack.com/Gavinp14/Gavinp14.github.io/3369-push-and-fetch-definitions/index.html#!./pages/vi/vi-github-and-repositories.md

### CODE REVIEW

File: pages/vi/vi-github-and-repositories.md [hunk 1]
```diff
@@ -63,6 +63,9 @@ Now, there are three levels of repositories to keep in mind:
 
 These repositories must be consistently synced and up-to-date with each other since we all contribute to the upstream repository (open-learning-exchange.github.io). It's crucial to keep changes separate and avoid mixing them between repositories. Significant differences can cause conflicts and prevent you from performing `git push/pull` operations smoothly.
 
+- **Push:** Sends your local changes to a remote repository.
+- **Fetch:** Retrieves updates from a remote repository without merging them into your local branch.
+
```
Comment (xyb994) [inline pages/vi/vi-github-and-repositories.md:68]: those are the expected changes

File: pages/vi/vi-github-and-repositories.md [hunk 2]
```diff
@@ -71,43 +74,43 @@ To fetch updates from the upstream repository, configure it as follows:
...
+This should show:
 
-  ```bash
-  origin  https://github.com/<YourUserName>/<YourUserName>.github.io.git (fetch)
-  origin  https://github.com/<YourUserName>/<YourUserName>.github.io.git (push)
-  ```
+```bash
+origin  https://github.com/<YourUserName>/<YourUserName>.github.io.git (fetch)
+origin  https://github.com/<YourUserName>/<YourUserName>.github.io.git (push)
+```
```
Comment (xyb994) [inline pages/vi/vi-github-and-repositories.md:92]: Are you using a linter that automatically applies rules when you save a file? I’m noticing changes unrelated to the issue, and comparing the two versions, the original one renders better with properly indented code blocks.

https://raw.githack.com/Gavinp14/Gavinp14.github.io/3369-push-and-fetch-definitions/index.html#!./pages/vi/vi-github-and-repositories.md#3._Configure_a_Remote_Repository_for_Your_Fork

https://open-learning-exchange.github.io/#!./pages/vi/vi-github-and-repositories.md#3._Configure_a_Remote_Repository_for_Your_Fork

**Review by xyb994 — CHANGES_REQUESTED** [review]

File: pages/vi/vi-github-and-repositories.md (hunk 2 above)
Comment (Gavinp14) [reply pages/vi/vi-github-and-repositories.md:92]: I will make these changes. I'm not sure why lines 77-92 are showing changes as I only added the push and fetch definitions.

**Comment by xyb994** [comment]
replaced by #3444

**Pull request closed without merging**